### DISCUSSION TITLE
Samsung Watch3 with Tizen

### DISCUSSION THREAD

**Discussion by nickrout** [Navid200/xDrip#28] [Watches]
I have a one day only chance to buy a Samsung Watch3. (no pressure)

This does not run WearOS. It runs Tizen, Samsungs own, apparently based on Android, OS.

Does anyone know if this can work with Xdrip?

**Comment by Navid200** [comment]
I am hoping the xDrip users and developers will use these forums so that we can have an alternative to the facebook group.  
Thanks for posting here.  

I personally have no experience with wear.
I can find this related to Tizen:  https://github.com/NightscoutFoundation/xDrip/issues/460

**Comment by szantos** [comment]
Sorry for the late reply, but may be useful for someone else, too.

With G-Watch App you can use xDrip as source and display data on your Samsung Tizen Watch.

See:
"G-Watch App Users" facebook group
"G-Watch App" on Google Play
